common/authen.js

```diff
@@ -154,6 +158,7 @@
                         // To avoid problems when user explicitly close the modal
                         modalInstance.result.then(onModalClose, onModalClose);
 
+                        // TODO: Attach .catch w/ catchAll
```

Comment:
Same comment as above:
This is unnecessary because the outer `.catch()` that should be added to [line 181](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/authen.js#L182) will catch an error from here.